<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarTextDisplay</class>
 <widget class="QWidget" name="ToolBarTextDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="1">
    <widget class="QCheckBox" name="lengthBox">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="authorBox">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="coordinatesBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Coordinates</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="diameterBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Pipe diameter</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="nameBox">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="phaseBox">
     <property name="text">
      <string>Phase</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="themeComboBox">
     <item>
      <property name="text">
       <string>Dark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Light</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="disciplineBox">
     <property name="text">
      <string>Discipline</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="identifierBox">
     <property name="text">
      <string>Identifier</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="indexBox">
     <property name="text">
      <string>Index</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Text color theme</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QDoubleSpinBox" name="fontSizeSpinBox">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>8.000000000000000</double>
     </property>
     <property name="maximum">
      <double>78.000000000000000</double>
     </property>
     <property name="value">
      <double>13.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Text font size</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>indexBox</tabstop>
  <tabstop>nameBox</tabstop>
  <tabstop>diameterBox</tabstop>
  <tabstop>authorBox</tabstop>
  <tabstop>disciplineBox</tabstop>
  <tabstop>lengthBox</tabstop>
  <tabstop>identifierBox</tabstop>
  <tabstop>phaseBox</tabstop>
  <tabstop>coordinatesBox</tabstop>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>themeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
